CHANGES.rst

```diff
@@ -29,29 +29,36 @@ CHANGES
 
     Example: if your test file looked like this ::
 
-        # my/package/tests/test_all.py
-        from zope.app.testing.functional import defineLayer
-        from zope.app.testing.functional import FunctionalDocFileSuite
-        defineLayer('MyFtestLayer', 'ftesting.zcml', allow_teardown=True)
+        # my/package/tests.py
+        from unittest import TestSuite
+        import doctest
+        import zope.app.wsgi.testlayer
+
+        layer = zope.app.wsgi.testlayer.BrowserLayer(my.package)
 
         def test_suite():
-            suite = FunctionalDocFileSuite('test.txt', ...)
-            suite.layer = MyFtestLayer
-            return suite
+            suite = doctest.DocFileSuite('test.txt', ...)
+            suite.layer = layer
+            return TestSuite((suite,))
 
```

Comment:
Um.  The point of this was to take the old-style config for those who use `defineLayer()`/`FunctionalDocFileSuite()` and tell them how to migrate to the new zope.testbrowser.

Now you're leaving those users out in the cold and instead are telling other users who already used `wsgi.testlayer` what to do.